### PR TITLE
Add readme check to docs tox environment

### LINKED ISSUE #74: Add readme check to Travis CI
Something similar to https://github.com/pypa/virtualenv/commit/18b374dc3eeb0e37aa3ad1b22c442712d9dca103

### PR DESCRIPTION
**Pull request by kragniz**
Closes #74.

### CODE REVIEW

**Comment by sigmavirus24** [comment]
So I'd rather have a separate testenv and a separate Travis job for this, e.g., 

- [.travis.yml](https://github.com/sigmavirus24/github3.py/blob/19481221d8e7741839646bb1b7cb77318566619a/.travis.yml#L50)
- [tox.ini](https://github.com/sigmavirus24/github3.py/blob/19481221d8e7741839646bb1b7cb77318566619a/tox.ini#L54)

Although you don't need to make this match github3.py's. What does the `-m` flag do?

**Comment by kragniz** [comment]
`-m` checks setup.py's metadata. It's not really related to a readme check, so I'll remove it.

File: .travis.yml [hunk 1]
```diff
@@ -40,5 +40,6 @@ matrix:
     - env: TOXENV=py27-flake8
     - env: TOXENV=py34-flake8
     - env: TOXENV=docstrings
+    - env: TOXENV=readme
```
Comment (sigmavirus24) [inline .travis.yml]: You could have kept `-env: TOXENV=docs` too. Or you can send a new PR with that :-D

Comment (kragniz) [inline .travis.yml]: You tempt me so...

Comment (kragniz) [inline .travis.yml]: #88 :sparkles:

**Pull request merged**